src/serial/xterm-keyboard.ts

```diff
@@ -12,6 +12,24 @@ const copyShortcut = (e: KeyboardEvent): boolean => {
   return true;
 };
 
+let escWasPressed = false;
+let escTimeout: NodeJS.Timeout;
```

Comment:
I think just type this as any - I've found these typings to be a real pain over time as sometimes you find NodeJS.Timeout is allowed and sometimes not depending on what you have installed.